### ISSUE TITLE
`npm run build` results in empty lib/css/_datepicker.css

### ISSUE THREAD

**Issue by angusfretwell** [airbnb/react-dates#1050]
Hi, I'm trying to put together a PR to add a class name for the `after-hovered-start` modifier (which seems like the best way to style the hovered minimum nights when hovering the selected start date, see #1025). 

I've got it working and want to test in my app, but can't get the CSS to build.

**Comment by majapw** [comment]
@angusfretwell can you rebase on the latest master and try again? You may be on one of the empty css creating patch versions... but it should be fixed now!

**Comment by angusfretwell** [comment]
@majapw Rebased, rebuilt, and it's still missing 😢

**Comment by majapw** [comment]
Are you running `npm run build` or `npm run build:css`? The former generates the lib/css/_datepicker.css file... the latter builds `css/styles.css` which powers the css storybook. Can you share a bit more of your process?

**Comment by angusfretwell** [comment]
@majapw I am running `npm run build`, is that correct?

**Comment by majapw** [comment]
Hmm, does `npm run storybook:css` show you styled components when you navigate to `localhost:6006` in the browser?

Can you try a fresh `npm install` (`rm -rf node_modules` and `npm install` after) and then try building again?

**Comment by angusfretwell** [comment]
Ahuh, that seems to have done the trick. Sorry, should have really thought to try that myself. 

Thanks for your help.

**Comment by ndbroadbent** [comment]
Hello, I just ran into this issue again when I was trying to test someone's fork in my app (#1678).

I initially installed the dependencies with `yarn install`, and then ran `yarn build` (which runs `npm run build:css -- --optimize`.) This gave me an empty file at `lib/css/_datepicker.css`.

I saw [this comment](https://github.com/airbnb/react-dates/issues/1647#issuecomment-494647478) about yarn not really being supported, so I tried to start from scratch with `npm`:

```
rm -rf node_modules
npm install
npm run build:css -- -o
```

Unfortunately this still gives me an empty file at `lib/css/_datepicker.css`.

I added this line near the end of `scripts/buildCSS.js`:

```
console.log({outputFilePath, CSS, formattedCSS})
```

Output:

```
{ outputFilePath: './lib/css/_datepicker.css',
  CSS: '',
  formattedCSS: '' }
```

Any idea why `compileCSS` is returning an empty string, and how I can get the CSS to build?

(In the meantime, I can just copy in the `./lib/css/_datepicker.css` from the published NPM package.)
